<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Devices</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_LoadFileFds">
    <property name="geometry">
     <rect>
      <x>811</x>
      <y>110</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose fds file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_OpenFileFds">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>571</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Open File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CriticalLength">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Critical length for creating the devices</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Quantities1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_40">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Required quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_13">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Choose a member and create device in each cell</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_14">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Member's ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_MembersId">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_41">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Required quantity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Quantities2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create folder directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CreateFolderFds">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>571</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Select Folder:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CreateFolderFds">
    <property name="geometry">
     <rect>
      <x>811</x>
      <y>60</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Quantities3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>471</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Choose a node to create detailed devices with weights</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_42">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Required quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>550</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_justdisplay_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Node's ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_NodesId">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
